fastlane_core/lib/fastlane_core/configuration/configuration.rb

```diff
@@ -54,11 +54,11 @@ def verify_value_exists
       # Make sure the given value keys exist
       @values.each do |key, value|
         next if key == :trace # special treatment
-
         option = option_for_key(key)
         if option
+          @values[key] = option.auto_convert_value(value)
```

Comment:
We do end up doing this twice in some cases because of the different entry points to this code and the fact that values can be loaded into a `config` after it is initialized. This code we added will only be run on values that are present at initialize and later fetch will run on all values,[ but only if they don't already have a value](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/configuration.rb#L232) so I think this is OK. What do you think @KrauseFx ?